src/running-servoshell.md

```diff
@@ -12,11 +12,12 @@ Use `--help` to list the available command line options:
 $ ./servo --help
 ```
 
-Use `--pref` to [configure Servo’s behaviour](https://github.com/servo/servo/blob/main/resources/prefs.json), including to enable experimental web platform features.
-For example, to run our [Conway’s Game of Life demo](https://demo.servo.org/experiments/webgpu-game-of-life/) with WebGPU enabled:
+Use `--pref` to configure Servo’s behaviour including to enable experimental web platform features. You can find the list of [preferences] in the Servo API documentation. For example, to run our [Conway’s Game of Life demo](https://demo.servo.org/experiments/webgpu-game-of-life/) with WebGPU enabled:
```

Comment:
I believe the guideline is to have [one sentence per line](https://github.com/servo/book/pull/38#discussion_r1865278794).

Reply:
Thanks. This is fixed now.